.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@NLSanyu , Please do not commit this file.

Reply:
I've removed it